integrationTests/testProcessorNodeWithCoordinator.go

```diff
@@ -99,23 +81,14 @@ func CreateProcessorNodesWithNodesCoordinator(
 				fmt.Println("error creating node coordinator")
 			}
 
-			blsHasher, _ := blake2b.NewBlake2bWithSize(hashing.BlsHashSize)
-			llsig := &multisig2.BlsMultiSigner{Hasher: blsHasher}
-
-			pubKeysMap := pubKeysMapFromKeysMap(ncp)
-			kp := ncp[shardId][i]
-			multiSigner, err := multisig.NewBLSMultisig(
-				llsig,
-				pubKeysMap[shardId],
-				kp.BlockSignSk,
-				kp.BlockSignKeyGen,
-				uint16(i),
-			)
+			multiSigner, err := createMultiSigner(*cp, shardId, i)
 			if err != nil {
-				fmt.Printf("error generating multisigner: %s\n", err)
+				log.Error("error generating multisigner: %s\n", err)
```

Comment:
no need for the extra "\n". Valid here and on the other 3 locations
Could have been: 
```GO
log.Error("error generating multisigner", "error", err)
```